assets/src/edit-story/panels/shared.js

```diff
@@ -36,16 +36,23 @@ const Group = styled.label`
 	display: flex;
 	align-items: center;
 	margin-bottom: 5px;
+	opacity: ${ ( { disabled } ) => disabled ? 0.7 : 1 };
 `;
 
-function InputGroup( { label, value, isMultiple, onChange, postfix } ) {
+export const ActionButton = styled.button`
+	color: ${ ( { theme } ) => theme.colors.mg.v1 };
+	font-size: 11px;
```

Comment:
I'd rather not define font colors on individual elements – we should handle these further up the stack somewhere, to enforce consistency.

Reply:
So, the thing is... with styled-components we aren't actually doing the C of CSS - there's no real cascading going on. The stylesheet is generated as a _one-class-per-element_ style of CSS.

Cascading can be done in code (but won't be reflected in the generated CSS) by "extending" existing styled objects (conceptually similar to @mixin rather than @extend in sass), and that makes sense for conceptual inheritance/cascading. But for something as simple as colors, just using the theme is best practice.

For reference, Material UI and it's React implementation, uses a [similar approach](https://github.com/mui-org/material-ui/blob/master/packages/material-ui/src/Badge/Badge.js#L43-L57) of a common color scheme defined in one place and then using these colors on individual components directly (albeit in a much more complex scenario). Generated CSS is in this case also not actually cascading.

Reply:
Yep, understood. I still greatly prefer the cascade personally, but willing to work with this for now. Let's extend the objects then, in this case, or do the same thing as colors and use a theme variable.